core/ordering/cosipbft/proof.go

```diff
@@ -40,9 +40,9 @@ func (p Proof) GetValue() []byte {
 }
 
 // Verify takes the genesis block and the verifier factory to verify the chain
-// up to the latest block.
+// up to the latest block. It verifies the whole chain.
```

Comment:
Indeed, and as such this function doesn't scale.
Is it a utility function that's never in the hot path or is this going to be our next bottleneck? 😅

Reply:
As long as we don't have skipchains with higher links, yes, this is going to be a bottleneck. But it should be significantly faster with the link/bocklink optimization.

Reply:
Right, I wasn't sure this was the same `GetProof` we were talking about yesterday. I must be getting old 😅 
Fair enough, we'll look at higher links Reasonably Soon (TM).